### ISSUE TITLE
Unable to Connect to Crashplan

### ISSUE THREAD

**Issue by LighthouseBZ** [jlesage/docker-crashplan-pro#375]
Im having a weird issue.

On my crashplan account dashboard it shows that my device is offline when its online. 

In the app it says not licensed - contact your administrator for assistance.

Its also not detecting my files and its telling me there is a newer version. I pulled a new docker instance but its the same. 

<img width="607" alt="Screen Shot 2022-05-13 at 2 10 56 PM" src="https://user-images.githubusercontent.com/73550628/168382685-0a6fcce0-12a1-409b-ba6e-b135533d316c.png">
<img width="749" alt="Screen Shot 2022-05-13 at 2 12 03 PM" src="https://user-images.githubusercontent.com/73550628/168382688-2b22de97-cd5c-4566-b3b0-4f7612c842a4.png">

<img width="698" alt="Screen Shot 2022-05-13 at 2 11 08 PM" src="https://user-images.githubusercontent.com/73550628/168382680-4b1d0fab-6024-469e-9801-519e46bf251f.png">
<img width="1110" alt="Screen Shot 2022-05-13 at 2 11 19 PM" src="https://user-images.githubusercontent.com/73550628/168382683-af9ca8bf-214b-4733-8d1b-d7ecc2677fb8.png">

**Comment by LighthouseBZ** [comment]
I also deleted the cache and restarted but it does nothing

**Comment by jlesage** [comment]
I'm working on integrating the latest version of CrashPlan.

But I'm not sure that the "Not licensed" message is related.  I guess that you verified that your subscription is fine ?

**Comment by LighthouseBZ** [comment]
Yes. My subscription just renewed. However, I did open a ticket with them
so waiting to hear back from them.

**Comment by jlesage** [comment]
A new image integrating the latest CrashPlan version is available.  You may want to update to see if you still have the license issue.

**Comment by JeroenAdam** [comment]
FYI, 2 days ago, I got an e-mail alert for each of our Crashplan instances saying "(servername) has not backed up in 1 day. I this computer hans't been uded in...)
I got Watchtower doing the latest update 9 hours ago, probably all will be fine.
I was wondering it this behavior is expected, struggling with these e-mail alerts for time to time.

**Comment by LighthouseBZ** [comment]
Updating to the latest version fixed the issue for me. Its synchronizing now.
